### PR TITLE
Fixed i1136 in EightSchoolsLogPDF

### PR DESCRIPTION
**Pull request by simonmarchant**
Added a term dL1[0] += (theta - mu) / tau**2 in evaluateS1 of EightSchoolsLogPDF, and appropriate tests.

### CODE REVIEW

File: pints/tests/test_toy_eight_schools_logpdf.py [hunk 1]
```diff
@@ -31,6 +31,15 @@ def test_basic(self):
         val = f([1, 0.5, 0.4, 1, 1, 1, 1, 1, 1, 1])
         self.assertAlmostEqual(val, -38.24061255483484, places=6)
 
+        # Default settings with non-ones input
+        f1, dp = f.evaluateS1(list(n + 1 for n in range(10)))
```
Comment (MichaelClerx) [inline pints/tests/test_toy_eight_schools_logpdf.py:35]: usually you'd just write `evaluateS1([n + 1 for ...])`

**Comment by MichaelClerx** [comment]
While we're at it: the math block for this class doesn't render properly:

https://pints.readthedocs.io/en/latest/toy/eight_schools_logpdf.html

File: pints/tests/test_toy_eight_schools_logpdf.py (hunk 1 above)
Comment (simonmarchant) [reply pints/tests/test_toy_eight_schools_logpdf.py]: Fixed in commit f866c8f

**Comment by simonmarchant** [comment]
@ben18785 I've made the changes Michael identified here. The math block should now render correctly but I'm not sure how to check it properly renders in sphinx/readthedocs.

**Comment by MichaelClerx** [comment]
Thanks!

**Comment by MichaelClerx** [comment]
> The math block should now render correctly but I'm not sure how to check it properly renders in sphinx/readthedocs.

See https://github.com/pints-team/pints/blob/master/CONTRIBUTING.md#building-the-documentation

**Comment by simonmarchant** [comment]
> Looks really good - thanks @simonmarchant! Can I just check -- did you test out how whether centered model sampling (in the example notebook) looks any different now? I don't suspect it would look significantly different.

You would be right, it looks very similar! I've pushed with an updated ipynb; the centred output looks like this (new version on left). One of the threads gets less stuck than before, otherwise it looks similar to me.
image

**Comment by ben18785** [comment]
@simonmarchant thanks -- looks great!

**Pull request merged**